apps/condo/domains/organization/integrations/sbbol/SbbolSecretStorage.js

```diff
@@ -38,31 +38,43 @@ class SbbolSecretStorage {
         await this.#setValue('clientSecret:updatedAt', dayjs().toISOString())
     }
 
-    async getAccessToken () {
-        return this.#getValue('accessToken')
+    async getAccessToken (userId) {
+        return this.#getValue(`user:${userId}:accessToken`)
     }
 
-    async setAccessToken (value, options) {
-        await this.#setValue('accessToken', value, options)
-        await this.#setValue('accessToken:updatedAt', dayjs().toISOString())
+    async setAccessToken (
+        value,
+        userId,
+        options,
+    ) {
+        if (!value ||
+            !userId
+        ) return logger.error('value and userId is required for setAccessToken')
+
+        await this.#setValue(`user:${userId}:accessToken`, value, options)
+        await this.#setValue(`user:${userId}:accessToken:updatedAt`, dayjs().toISOString())
     }
 
-    async isAccessTokenExpired () {
-        return await this.#isExpired('accessToken')
+    async isAccessTokenExpired (userId) {
+        return await this.#isExpired(`user:${userId}:accessToken`)
     }
 
-    async getRefreshToken () {
-        return this.#getValue('refreshToken')
+    async getRefreshToken (userId) {
+        return this.#getValue(`user:${userId}:refreshToken`)
     }
 
-    async setRefreshToken (value) {
+    async setRefreshToken (
+        value,
+        userId,
+    ) {
+        if (!value || !userId) return logger.error('value and userId is required for setRefreshToken')
```

Comment:
One condition for one variable, please. This way it's easier to figure out where exactly the problem occurs
Throw error, instead of logger, please